getting-started/templates/systemlink-values.yaml

```diff
@@ -503,6 +503,40 @@ dataframeservice:
       ## The name of the key in the above secret whose value contains the Dremio password
       ##
       passwordKey: "password"
+    distStorage:
```

Comment:
> If users need to make changes to their deployment, make sure this is tracked in the release notes.

@JD-Robertson, maybe you can help me understand if we consider this a breaking change or not. The DFS and umbrella chart default values files don't enable distributed storage, so if GM doesn't merge in the new systemlink-values.yaml changes, they won't need to do anything. If they do merge it in, they will need to resolve all of the <ATTENTION> call-outs that I've added.

Another thing that I should disclose is that the Dremio chart will fail `helm lint` and `helm template` unless users pass in both systemlink-values.yaml and systemlink-secrets.yaml as arguments when deploying. I was under the impression that customers were already specifying both values files when deploying, but let me know if I'm mistaken or if we're not comfortable with introducing this dependency.

Reply:
We have guidelines for what requires release notes: https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/docs/staging/Updating%20the%20Release%20Notes%20for%20SystemLink%20Enterprise.md

I don't expect customers to merge changes to this template with their existing values files. However, if there is functionality that is marked with <ATTENTION> because the user needs to decide whether to opt into or out of it then it should be noted in the release notes so that upgrading users are able to make a decision about this functionality. Specifically, here, if we expect a customer to require the Dremio improvements this provides then we should call it out.

Reply:
I added something to the spreadsheet, and I'll update it when the installation guide change is made. Agreed with your take that this isn't a breaking change, but it is something that we really want users to opt into.